variables.tf

```
@@ -46,6 +46,18 @@ variable "database_password" {
   description = "Password for the primary DB user. Required unless a `snapshot_identifier` or `replicate_source_db` is provided."
 }
 
+variable "database_manage_master_user_password" {
+  type        = bool
+  default     = null
```

I feel like the `default` should be `false` instead of `null` because even though they should accomplish the same thing, `false` is more clear and keeps in line with `bool` arguments being binary. 
```suggestion
  default     = false
```

---

changed